modules/maintaining-bare-metal-hosts.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/managing-bare-metal-hosts.adoc
+
+[id="maintaining-bare-metal-hosts_{context}"]
+= Maintaining bare metal hosts
+
+You can maintain the details of the bare metal hosts in your cluster from the {product-title} web console. Navigate to *Compute* -> *Bare Metal Hosts*, and select a task from the *Actions* drop down menu. Here you can manage items like BMC details, boot MAC address for the host, enable power management, etc. Details of the network interfaces and disks for the host can be reviewed.
```

Comment:
Last sentence is passive (can be reviewed), and I think in Linux we prefer to use drive rather than disk.